src/Type/ArrayType.php

```diff
@@ -12,21 +12,21 @@ class ArrayType implements StaticResolvableType
 	/** @var bool */
 	private $itemTypeInferredFromLiteralArray;
 
-	/** @var bool */
-	private $possiblyCallable;
+	/** @var TrinaryLogic */
+	private $callable;
```

Comment:
This commit should bump the cache version in FileTypeMapper because of the change in ArrayType.